mu-plugins/blocks/global-header-footer/blocks.php

```diff
@@ -183,6 +195,60 @@ function restore_inner_group_container() {
 	}
 }
 
+/**
+ * Render the global header via a REST request.
+ *
+ * @return string
+ */
+function rest_render_global_header( $request ) {
+
+	$markup = render_global_header();
+
+	// Serve the request as HTML
+	add_filter( 'rest_pre_serve_request', function( $served, $result ) {
+		header( 'Content-Type: text/html' );
+
+		echo $result->get_data();
+
+		return $served = true;
```

Comment:
Why this instead of just `return true` ?

Won't `$foo = true` always evaluate to `true`?

Reply:
Yes. It's more a coding style I've been in the habit of in filters to make it clear to myself that I'm handling the filter correctly.

Happy to switch to return true directly.

```suggestion
		return true;
```

Reply:
Ah, I see 👍🏻  I'm not sure it'll be obvious to others, but I don't feel strongly

Reply:
GitHub doesn't show it clearly in this thread view, but I committed the above suggestion shortly after commenting. https://github.com/WordPress/wporg-mu-plugins/pull/62/commits/d70f783ed1f91723f332a4c00c962fa724bc24fa